source/pug/templates/base-for-django.pug

```diff
@@ -8,6 +8,10 @@ block masterParams
   - let isMezzanine = true
   - let pageTitle = `{{ page.meta_title }}`;
 
+block metaTags
```

Comment:
we updated the pug files already so that there's a `socialMetadata` block for this now